src/modules/orders/orders.service.ts

```diff
@@ -65,6 +66,68 @@ export class OrdersService {
     }
   }
 
+  async findByUserIdAndOrderId(
+    userId: string,
+    orderId: number,
+  ): Promise<SingleOrderResponse> {
+    try {
+      const user = await this.userRepository.findOne({
+        where: { id: userId },
+      });
```

Comment:
I assume you do this when you validate token, no?

Reply:
You are right, removed the unnecessary db call